tf2/src/buffer_core.cpp

```diff
@@ -113,8 +113,13 @@ CompactFrameID BufferCore::validateFrameId(
     return 0;
   }
 
+  // Only check that a frame is non-existent once a frame was found
+  static bool init_state = true;
```

Comment:
A static variable here is not correct. It will not correctly interact with potentially multiple BufferCore instances and doesn't deal with resetting the core.